app/views/record/_availability.html.erb

```diff
@@ -10,7 +10,7 @@
           <a class="button button-secondary" href="<%= login_url %>">Sign in for access</a>
         </div>
       </div>
-    <% else %>
+    <% elsif relevant_fulltext_link?(@record.fulltext_link) %>
```

Comment:
Should we be performing this check earlier, where we now have `<% if @record.fulltext_link.present? %>`? (Meta-question: could it happen that there's an irrelevant fulltext link, and we offer "sign in for access", and they sign in, and then are presented with no access option?)

Reply:
I'm pretty sure (i.e. I hope) the logic behind the previous condition only triggers for pdflinks which are always relevant and always require non-guest... so I think we're good.